src/D2L.Bmx/ParameterDescriptions.cs

```diff
@@ -18,4 +18,7 @@ internal static class ParameterDescriptions {
 		Write BMX command to AWS profile, so that AWS tools & SDKs using the profile will source credentials from BMX.
 		See https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sourcing-external.html.
 		""";
+
+	public const string ExperimentalBypassBrowserSecurity = "Enables experimental features";
+	public const string Passwordless = "Use Okta DSSO to attempt to authenticate without providing a password";
```

Comment:
Definitely don't say "DSSO" in any user-facing message. Nobody knows what it means.